test/e2e/config/packet-ci-actions.yaml

```diff
@@ -6,7 +6,7 @@
 # - packet
 
 images:
-- name: "${REGISTRY:=ghcr.io}/${IMAGE_NAME:=kubernetes-sigs/cluster-api-provider-packet}:${TAG:=e2e}"
+- name: "${REGISTRY:=ghcr.io/equinix-labs}/${IMAGE_NAME:=cluster-api-provider-packet}:${TAG:=e2e}"
```

Comment:
I think this image name change should be reverted

Reply:
Yep, good catch.